src/swar_synthesizer.py

```diff
@@ -4,82 +4,98 @@
 import struct
 import pyaudio
 
-def swar2freq(swar,basefreq):
+SWAR_FREQUENCY_MAPPER = {
```

Comment:
Wondering if we should move `SWAR_FREQUENCY_MAPPER` and `VALID_SWAR_INPUT` subsets into a separate notes.txt or so since they're constants.

Reply:
Keeping them as a global dict and set (respectively) is better than loading them from a text / JSON file as the latter involves loading of files which in turn involves more computation (comparitively) than the former. 😅